### DISCUSSION TITLE
Positional Audio Support

### DISCUSSION THREAD

**Discussion by ReynoldsCahoon** [stoatchat/discussions#195] [Feature Suggestions]
[Teamspeak](https://business.teamspeak.com/features/3d-audio-example/) and [Mumble](https://wiki.mumble.info/wiki/Positional-Audio) both offer positional audio support for games and applications to support. This allows for very immersive experiences.

This feature would provide functionality that Discord doesn't support.

**Comment by The-Starflower** [comment]
if we could use plugins like acre and tfar (Arma 3 radio mods)
tha would be actully pretty amazing

**Comment by dannyhpy** [comment]
That feature alone would be game-changing

**Comment by minkanjin** [comment]
I think the best way to do this is to copy Mumble's code as closely as the license allows. If it is 1to1 compatible with Mumble, then either Revolt or Mumble can be used at any time. And this would be great for game devs, because they can implement support for two platforms with one library import. Same for after market hackers who want to add the feature to a game. This would make it easiest for everyone, which might help Revolt uptake a bit.

**Reply by minkanjin** [reply]
Maybe the Mumble guys can be convinced to move that code to a separate library
